### PR TITLE
Dylan branch

### PR DESCRIPTION
**Pull request by dylanpizzo**

### CODE REVIEW

**Comment by dylanpizzo** [comment]
Be careful when merging from dev back into master. I updated the favicon.ico file to be a dev version to fix issue with dev favicon loading and i put favicon.ico in gitignore (along with logo512.png, etc.). So it probably shouldn't merge those into master, hopefully. Idk though. Just beware.

File: harmonyfrontend/src/components/library/series-list.jsx [hunk 1]
```diff
@@ -49,8 +50,12 @@ export default observer(
         return <Loading />;
       }
 
+      let seriesTitles = Object.keys(series).sort((a, b) => {
+        return sortableTitle(a) > sortableTitle(b) ? 1 : -1;
```
Comment (garethgeorge) [inline harmonyfrontend/src/components/library/series-list.jsx]: I like the idea of using an intelligent title sort, this is great but @dylanpizzo the code should be moved src/model/library.js wth a new function called 
```
sortTitles(titles)
```
or something of that sort which takes an array of strings and returns them sorted using the title sort rules you define.

File: harmonyfrontend/src/components/library/movies-list.jsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { observer } from "mobx-react";
 import Loading from "../../components/loading";
 import Movie from "./movie";
 import "./library.scss";
+import sortableTitle from "./titlesort";
```
Comment (garethgeorge) [inline harmonyfrontend/src/components/library/movies-list.jsx]: @dylanpizzo same comment as below, not a fan of a file in the /components/ directory which exports a function that isn't a component. Should instead be a utility method added to the model.

**Comment by garethgeorge** [comment]
@dylanpizzo looks good for the most part but I have a few comments on the title sorting, will hold off on merging until we meet up this weekend :)

**Pull request merged**